scripts/install_plugins.sh

```diff
@@ -5,32 +5,84 @@ CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source "$CURRENT_DIR/shared_functions.sh"
 
 clone() {
-	local plugin=$1
-	cd $SHARED_TPM_PATH &&
+	local plugin; plugin="$1"
+	cd "$SHARED_TPM_PATH" &&
```

Comment:
Agree @jbnicolai =), actually I've been thinking in creating a `cleanup` branch or something, while reviewing the code I've seen a couple of details and sometimes I compulsively just edit them